rules-reviewed/jboss-eap/weblogic/weblogic-catchall.windup.xml

```diff
@@ -28,18 +27,19 @@
                         </not>
                     </when>
                     <perform>
-                        <classification title="Oracle" severity="mandatory" />
+                        <classification title="Oracle Weblogic" severity="mandatory" />
                         <hint effort="1" severity="mandatory" title="Oracle proprietary type reference">
                             <message>
                                 This is an Oracle proprietary type and needs to be migrated to a compatible API. There is currently no detailed
                                 information about this type.
                             </message>
+                            <tag>catchall</tag>
                         </hint>
                     </perform>
                 </iteration>
             </perform>
             <where param="package">
-                <matches pattern="(com\.)?(weblogic|oracle)(\..*)?" />
+                <matches pattern="(com\.)?(weblogic|oracle|bea|tangosol|sonic|oracle\.sql)(\..*)?" />
```

Comment:
you are right, it is basically Progress/Fuse. I will move this one to Fuse rules.